account_credit_control/views/credit_control_line.xml

```diff
@@ -118,6 +111,12 @@
                     <filter domain='[]'
                             context="{'group_by': 'manually_overridden'}"
                             icon="terp-document-new" string="Manual change"/>
+                    <separator orientation="vertical"/>
+                    <field name="partner_id"/>
+                    <field name="account_id"/>
+                    <field name="invoice_id"/>
+                    <field name="level"/>
+                    <field name="policy_level_id"/>
```

Comment:
Those are not group_by, the name="group_by" was probably wrong. AFAIK only `filter` with `group_by` in context are actually group by